cli/pkg/version/version_checker.go

```diff
@@ -129,7 +129,9 @@ func (v *VersionChecker) getNewerCLIVersion(usedVersionString string) (available
 	return res, nil
 }
 
-const newCompatibleVersionMsg = `* Keptn CLI version %s is available! Please visit https://keptn.sh/docs/%s/operate/upgrade/ for more information.`
+const newCompatibleVersionMsg = `* Keptn CLI version %s is available! 
+ - To update to the latest CLI run this command: curl -sL https://get.keptn.sh | sudo -E bash
```

Comment:
Please use the `curl ...` command without `sudo -E`, as shown in our [Quickstart](https://keptn.sh/docs/quickstart/#2-1-install-keptn-cli).

Reply:
@christian-kreuzberger-dtx my bad, I presumed sudo would be required to mv the binary to /usr/local/bin, I have fixed it!

Reply:
It is, but we moved it inside the get.keptn.sh script ;) thanks for the fix